### ISSUE TITLE
do not encode card order in the first line of a card

### ISSUE THREAD

**Issue by manogna4** [imdone/imdone#73]
**Is your feature request related to a problem? Please describe.**
both the `HASHTAG` and the `MARKDOWN` format encode card order in the first line of the card.

While this works fine in the imdone UI, it complicates a very important line in the markdown journal with something that is not primary to the card. Further, the string representing the number can get quite large as you move stuff around because it can utilize several decimal places (I believe I've seen cases of 6+ decimals).

This makes the markdown file less legible with extended usage.

**Describe the solution you'd like**
I think a better place to encode this information would be the comment section at the end of a card, where `created:<datetime>` is inserted (if you don't change default options).

so, it should be something like this:

```html
<!--
created:2021-03-13T12:44:07.871Z #automatically
card-order:10
-->
```

I think this is more appropriate since the card order value is not meant to be handled manually by the user anyway.

**Describe alternatives you've considered**
all available alternatives involve the card order being put in the first line of the card.

**Additional context**
this applies to card generated by markdown files (and not code files)

**Comment by jtmusky** [comment]
>  Further, the string representing the number can get quite large as you move stuff around because it can utilize several decimal places (I believe I've seen cases of 6+ decimals).

I've see this go as far as 10 x 10^-11.

What if instead of 10base, use 16base? 0000-FFFF, I can't see having more than 65k cards. e.g. card-order:3A
Oh idea, what about also using "." designate child/dep cards? e.g. (I'm just brain storming here, usually with only one cloud, ha)

```
[# Do this thing Parent](#TODO:)
card-ord:3A

[# Do this thing Child/First](#TODO:)
card-ord:3A.1

[# Do this thing Child/First](#TODO:)
card-ord:3A.2
```

What about using an ID for the card AND order?

The order could state the dep?
```
[# Do this thing Parent](#TODO:3A)
card-ord:2D8

[# Do this thing Child/First](#DOING:3B)
card-ord:C.3A

[# Do this thing Child/First](#DOING:3C)
card-ord:5.3B
```
Maybe put child card's in the id? `#DOING:3A.2`

idk ¯\\_(ツ)_/¯

**Comment by piascikj** [comment]
Implemented in 1.17.3
